charts/spire/charts/spiffe-csi-driver/values.yaml

```diff
@@ -0,0 +1,67 @@
+image:
```

Comment:
Should we group the `spiffe-csi-driver` values in a `spiffe-csi-driver `sub section (e.g. like nodeDriverRegistrar) ?

I don't have a preference myself. I think I have seen both styles.

Reply:
For use from the toplevel chart it will already be:

```yaml
spiffe-csi-driver:
	image: …
    nodeDriverRegistrar: …
```